package/rmkit/package

```diff
@@ -62,7 +62,7 @@ nao() {
 remux() {
     pkgdesc="App launcher that supports multi-tasking applications"
     url="https://rmkit.dev/apps/remux"
-    pkgver=0.1.0-1
+    pkgver=0.1.0-2
```

Comment:
No version number change?

Reply:
i don't have internal version numbers for programs in rmkit,  I'll add that to the todo list